### PR TITLE
Initial commit to add API to retrieve min throughput for a CosmosContainer

### PR DESCRIPTION
**Pull request by abinav2307**
This changeset adds an API to the CosmosContainer to allow users to fetch the minimum throughput that a container in Cosmos DB can be scaled down to.

### CODE REVIEW

File: sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosContainer.java [hunk 1]
```diff
@@ -427,6 +427,25 @@ public CosmosConflict getConflict(String id) {
...
 
+    /**
+     * Gets the min throughput to which this container can be scaled down to
+     * 
+     * @return a {@link Mono} containing min throughput or an error.
+     */
+    public Mono<Integer> readMinThroughput() {
+        return this.read().flatMap(cosmosContainerResponse -> database.getDocClientWrapper()
+            .queryOffers("select * from c where c.offerResourceId = '"
+                    + cosmosContainerResponse.resourceSettings().resourceId() + "'", new FeedOptions())
```
Comment (moderakh) [inline sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosContainer.java]: susceptible to SQL injection attack.

Comment (abinav2307) [reply sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosContainer.java]: Fixed in latest iteration

**Comment by abinav2307** [comment]
@kushagraThapar - tests added.

**Comment by kushagraThapar** [comment]
The builds are failing for some other reason, I am working with Azure Engineering team to get them fixed. We will merge this PR after the builds are done.

**Pull request merged**